src/calculateRentalCost.js

```diff
@@ -28,7 +28,13 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  if (days < 3) {
+    return days * 40;
+  } else if (days < 7) {
+    return days * 40 - 20;
+  }
+
```

Comment:
look checklist one more time please
https://github.com/mate-academy/js_task-transportation-on-vacation/blob/master/checklist.md
1st - add please variables for main values: rent cost, discount, etc..